cpp/ps0000/ps0000_soln.cpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream> // 2/2
+
+using namespace std;
+
+int main() // 1/1
+{
+  cout << "Hello, World!\n"; // 2/4
+  return 0; // 1/1
+}
+// good work 👍 6/10
```

Comment:
- [x] most of code errors and bugs are fixed
- [ ] file is in the right path
- [ ] +1 mark for your first commit